base_comment_template/README.rst

```diff
@@ -1,11 +1,15 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=======================
 Base Comments Templates
 =======================
 
 Add a new model to define templates of comments to print on
 documents.
 
 Two positions are available for the comments:
-
 - above document lines
```

Comment:
put * instead of - for a better readability

Reply:
Done